src/AppBundle/Tests/Controller/AdmissionControllerTest.php

```diff
@@ -77,4 +91,34 @@ private function createAndSubmitForm(string $testname, bool $wantsNewsletter)
 
         $clientAnonymous->submit($form);
     }
+
+    /**
+     * @param bool $teamInterest
+     */
+    private function createAndSubmitForm_teamInterest(bool $teamInterest)
+    {
+        $clientAnonymous = static::createClient(array(), array(
+            'PHP_AUTH_USER' => 'assistent',
+            'PHP_AUTH_PW' => '1234',
+        ));
+
+        $crawler = $clientAnonymous->request('GET', '/opptak/eksisterende');
+        $this->assertTrue($clientAnonymous->getResponse()->isSuccessful());
```

Comment:
Siden du bruker `BaseWebTestCase` kan dette:
```php
$clientAnonymous = static::createClient(array(), array(
    'PHP_AUTH_USER' => 'assistent',
    'PHP_AUTH_PW' => '1234',
));
 
$crawler = $clientAnonymous->request('GET', '/opptak/eksisterende');
$this->assertTrue($clientAnonymous->getResponse()->isSuccessful());
```
nå skrives som:
```php
$crawler = $this->anonymousGoTo('/opptak/eksisterende');
```

Hvis man trenger å få tak i `Client` igjen (som man trenger i dette tilfellet for å submitte form litt lengre ned) får man den ved:
```php
$client = $crawler->getClient();
```